Revert "Revert "refactor: update proguard documentation"" (#348)
ZupIT/docs-beagle: content/en/resources/customization/beagle-for-android/proguard-configuration.md

## Introduction

> Beagle contains minimum rules necessary for its components to work without the need for your application to add any rules in ProGuard / R8. However, in the annotations `RegisterWidget` and ` RegisterAction` you need to fill them with their respective names and if you have an object inside your component you need to annotate it with `@ BeagleJson`

Reviewer: ```suggestion
Beagle contains minimum rules necessary for its components to work without the need to add, on your application, any rules in ProGuard / R8. 

To make sure these rules work, you must fill the annotations `RegisterWidget` and ` RegisterAction` with their respective names. In case you have an object inside your component, you need to annotate it with `@ BeagleJson`
```

## Introduction

> Beagle contains minimum rules necessary for its components to work without the need to add, on your application, any rules in ProGuard / R8. 
> 
> To make sure these rules work, you must fill the annotations `RegisterWidget` and ` RegisterAction` with their respective names. In case you have an object inside your component, you need to annotate it with `@ BeagleJson`

Reviewer 2: ```suggestion
Beagle contains necessary rules for its components to work without having to add on your application, any kind of rules in ProGuard / R8. 

In order to make sure these rules work, you must fill the annotations `RegisterWidget` and ` RegisterAction` with their names. If you have an object inside your component, you need to annotate it with `@ BeagleJson`
```

## Example

> You must note the classes used with the annotation `@BeagleJson` to make sure that your component works normally with ProGuard / R8 active.
> 

Reviewer 2: ```suggestion
You have to note the classes used with `@BeagleJson` annotation, to make sure that your component works normally with ProGuard / R8 active.

```

Reviewer 3: ```suggestion
You have to annotate the classes used with the `@BeagleJson`  annotation  to make sure that your component works normally with ProGuard / R8 active.

```
